.travis.yml

```diff
@@ -21,4 +21,4 @@ script:
     - make VERBOSE=1
 
 after_success:
-    - ./tests/test_hsluv
+    - make VERBOSE=1 CTEST_OUTPUT_ON_FAILURE=1 test
```

Comment:
This change makes imho the build log much less usable.

Compare the end of https://travis-ci.org/github/hsluv/hsluv-c/jobs/668877272 (master) and https://travis-ci.org/github/hsluv/hsluv-c/jobs/668875475 (this PR) (you may need to click the triangle icon at the left to roll down the output of the test command).

I'm afraid if anything goes wrong it would now say only something like "test_hsluv has failed." but not showing its output.

Can its output be written down as well with some options? If not, I would likely prefer to drop this one change.

Reply:
> This change makes imho the build log much less usable.

I see.

> I'm afraid if anything goes wrong it would now say only something like "test_hsluv has failed." but not showing its output.

I understand that fear. `CTEST_OUTPUT_ON_FAILURE=1` takes care of that in the Travis integration.

> Can its output be written down as well with some options? If not, I would likely prefer to drop this one change.

Let me demo `ARGS=--verbose` to you as an alternative approach, pushed.